lib/packages/docutils/rstgen.nim

```diff
@@ -1506,7 +1506,8 @@ proc rstToHtml*(s: string, options: RstParseOptions,
   initRstGenerator(d, outHtml, config, filen, options, myFindFile,
                    rst.defaultMsgHandler)
   var dummyHasToc = false
-  var rst = rstParse(s, filen, 0, 1, dummyHasToc, options)
+  var rst = rstParse(s, filen, line=LineRstInit, column=ColRstInit,
```

Comment:
the line error reporting is incorrect:
```
1. hello 1
1. hello 2
1. hello 3
zook
```

gives:
```
t11995.rst(3, 1) Warning: RST style:
  not enough indentation on line 4
      (if it's continuation of enumeration list),
  or no blank line after line 3 (if it should be the next paragraph),
  or no escaping \ at the beginning of line 3
      (if lines 3..4 are a normal paragraph, not enum. list) [warnRstStyle]

```
I'd expect:
t11995.rst(4, 1) Warning: RST style:
not
t11995.rst(3, 1) Warning: RST style:

ok to leave as TODO for future work though